libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -156,14 +156,22 @@ bool GCS_MAVLINK::init(uint8_t instance)
         return false;
     }
     
-    if (mavlink_protocol == AP_SerialManager::SerialProtocol_MAVLink2) {
+    if (mavlink_protocol == AP_SerialManager::SerialProtocol_MAVLink2 ||
+        mavlink_protocol == AP_SerialManager::SerialProtocol_MAVLinkHL) {
```

Comment:
This makes me wonder if we shouldn't use a `SERIAL_OPTION` bit instead of a separate serial protocol.  I'm really not sure, however.

Reply:
I did initially write this using a new ``SERIAL_OPTION`` bit. In the end, I decided a protocol would be more appropriate, as I have a future plan to add support for specific satellite modems. It'll be easier to figure out which links are applicable, as I can just search for ``SerialProtocol_MAVLinkHL``, rather than ``SERIAL_OPTION`` _and_ ``SerialProtocol_MAVLink2``